test/plugin/base_test.rb

```diff
@@ -40,6 +40,24 @@ def setup
   # generic attributes
   HOSTNAME = Socket.gethostname
 
+  WRITE_LOG_ENTRIES_URI = 'https://logging.googleapis.com/v2beta1/entries:write'
+
+  APPENGINE_SERVICE_NAME = 'appengine.googleapis.com'
```

Comment:
Changed. The full path is pretty long, so I assign them to local constants at the beginning of the file. Seems there is no way to import a class in a module in Ruby.